packages/shared_preferences/shared_preferences/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/flutter/plugins/tree/master/packages/shared_prefere
 # 0.5.y+z is compatible with 1.0.0, if you land a breaking change bump
 # the version to 2.0.0.
 # See more details: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
-version: 0.5.13+2
+version: 0.6.0-nullsafety
```

Comment:
I almost forgot (and may have missed this for other plugins): we actually need to make this 2.0.0-nullsafety. The README currently says that we guarantee 1.0 will be compatible with the current version, and encourages people setting their constraints to <2, but this is a breaking change.

(This should also update the README to remove that section.)